app/assets/stylesheets/sessions.scss

```diff
@@ -1,3 +1,7 @@
 // Place all the styles related to the Sessions controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+#or {
```

Comment:
I wonder if this should be `.or` rather than an id. I can imagine wanting to have this on the page more than once.